guardrails/guard.py

```diff
@@ -125,8 +131,22 @@ def __call__(
             The raw text output from the LLM and the validated output.
         """
         with start_action(action_type="guard_call", prompt_params=prompt_params):
+            if prompt:
+                if isinstance(prompt, str):
+                    assert (
```

Comment:
Is this for debugging? If we want to fail here, if the conditions are not met, may be rasing an exception would be better?